src/client/api/v1/get_balance.c

```diff
@@ -84,9 +84,9 @@ int deser_balance_info(char const *const j_str, res_balance_t *res) {
   return ret;
 }
 
-int get_balance(iota_client_conf_t const *conf, char const addr[], res_balance_t *res) {
+int get_balance(iota_client_conf_t const *conf, addr_type_enum addr_type, char const addr[], res_balance_t *res) {
```

Comment:
it accepts either bech32 or ed25519 format, let's keep it simple, true for bech32.

```suggestion
int get_balance(iota_client_conf_t const *conf, bool is_bech32, char const addr[], res_balance_t *res) {
```

Reply:
Done.